aws-sdk-resources/lib/aws-sdk-resources/services/s3/encryption/kms_cipher_provider.rb

```diff
@@ -25,7 +25,7 @@ def encryption_cipher
           envelope = {
             'x-amz-key-v2' => encode64(key_data.ciphertext_blob),
             'x-amz-iv' => encode64(cipher.iv = cipher.random_iv),
-            'x-amz-cek-alg' => 'AES/CBC/PKCS5Padding',
+            'x-amz-cek-alg' => 'AES/CBC/PKCS5Padding', #TODO allow selection
```

Comment:
Let's take out any TODO comments, as a style matter - I agree we should add this (although that isn't a blocker to me for accepting this PR, given the bug related to this), but we can handle that via other means.